### PR TITLE
patch: allow multiple screenshots w/o overwriting

### PR DESCRIPTION
**Pull request by TheoCGaming**
code::blocks ide also got rid of a bunch of trailing spaces because I tried to modify it in there but used a text editor anyway

### CODE REVIEW

**Comment by TheoCGaming** [comment]
there
it now has a timestamp to guarantee that it doesn't overwrite previous screenshots

File: mandelgpu.cpp [hunk 1]
```diff
@@ -319,29 +319,33 @@ int main(int argc, char* argv[])
...
     case 'f':
       gl_renderer::instance().toggle_fullscreen(!gl_renderer::instance().is_fullscreen());
       break;
     case 'p':
       // Write image
-      gl_renderer::instance().save_png_screenshot("mandelgpu.png");
+      std::string png1 = "mandelgpu-";
+      std::string pngIndex = std::to_string(std::time(0));
+      std::string png2 = ".png";
+      gl_renderer::instance().save_png_screenshot(png1 + pngIndex + png2); // MARKED HERE ------------------------------------------------------------------
```
Comment (illuhad) [inline mandelgpu.cpp]: What is the meaning of the "Marked here" comment? I don't think we need, so can we just remove it?

```suggestion
      gl_renderer::instance().save_png_screenshot(png1 + pngIndex + png2);
```

**Review by illuhad — COMMENTED** [review]
Good idea! Just one question :)

File: mandelgpu.cpp (hunk 1 above)
Comment (TheoCGaming) [reply mandelgpu.cpp]: oh yeah, remove that, I marked it there to remember where the line I needed to mess with was

**Comment by illuhad** [comment]
Thanks!

**Pull request merged**